src/test/mochitest/head.js

```diff
@@ -388,13 +388,12 @@ function isPaused(dbg) {
   return !!isPaused(getState());
 }
 
-async function waitForLoadedObjects(dbg) {
-  const { hasLoadingObjects } = dbg.selectors;
-  return waitForState(
-    dbg,
-    state => !hasLoadingObjects(state),
-    "loaded objects"
-  );
+async function waitForLoadedScopes(dbg) {
+  await waitUntil(() => findElement(dbg, "scopes"));
+  const scopes = findElement(dbg, "scopes");
```

Comment:
this can be `const scopes = await waitUntil(() => findElement(dbg, "scopes"));`

Reply:
mh, I'll try, but I looked up the waitUntil function, and it was resolving with true, not the result of the condition.
Though, I did not try what you suggest here, so I will

Reply:
this makes test fail because `scopes.querySelector` is not a function.

This is explained by https://searchfox.org/mozilla-central/rev/2031c0f517185b2bd0e8f6f92f9491b3410c1f7f/devtools/client/framework/test/shared-head.js#453-462 , where waitUntil resolves with `true`.

Reply:
oh, i think we have `waitForElement` :)